src/test/ui/type/issue-100584.stderr

```diff
@@ -0,0 +1,22 @@
+error: unused variable: `xyza`
+  --> $DIR/issue-100584.rs:2:8
+   |
+LL | fn foo(xyza: &str) {
+   |        ^^^^
+   |        |
+   |        unused variable
+   |        help: if this is intentional, prefix it with an underscore: `_xyza`
+LL |
+LL |     let _ = "{xyza}";
+   |             -------- you might have meant to use string interpolation in this string literal
```

Comment:
since this was already figured out, maybe make it a structured suggestion to suggest adding `format!(` before and `)` after the string literal.